services/subscriber/service.go

```diff
@@ -285,7 +293,8 @@ func (s *Service) updateSubs(wg *sync.WaitGroup) error {
 				}
 				sub, err := s.createSubscription(se, si.Mode, si.Destinations)
 				if err != nil {
-					return err
+					s.Logger.Println("createSubscription failed for:", si.Name)
```

Comment:
Again, shouldn't this be an error? Why would it be sensible to continue after encountering this?